android/res/layout/dialog_download_single_item.xml

```diff
@@ -1,34 +1,34 @@
 <?xml version="1.0" encoding="utf-8"?>
 <LinearLayout
-    xmlns:android="http://schemas.android.com/apk/res/android"
-    android:layout_width="wrap_content"
-    android:layout_height="wrap_content"
-    android:orientation="vertical">
+  xmlns:android="http://schemas.android.com/apk/res/android"
+  android:layout_width="wrap_content"
+  android:layout_height="wrap_content"
+  android:orientation="vertical">
 
   <TextView
-      android:id="@+id/tv__message"
-      android:layout_width="wrap_content"
-      android:layout_height="wrap_content"
-      android:layout_marginBottom="@dimen/margin_base"
-      android:paddingLeft="@dimen/margin_base_plus"
-      android:paddingRight="@dimen/margin_base_plus"
-      android:textAppearance="@style/MwmTextAppearance.Body2"/>
+    android:id="@+id/tv__message"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:layout_marginBottom="@dimen/margin_base"
+    android:paddingLeft="@dimen/margin_base_plus"
+    android:paddingRight="@dimen/margin_base_plus"
+    android:textAppearance="@style/MwmTextAppearance.Body2"/>
 
   <View
-      android:layout_width="match_parent"
-      android:layout_height="1dp"
-      android:background="?attr/dividerHorizontal"/>
+    android:layout_width="match_parent"
+    android:layout_height="1dp"
+    android:background="?attr/dividerHorizontal"/>
```

Comment:
PTAL
> On Apr 5, 2016, at 3:29 PM, Alexander Marchuk <notifications@github.com> wrote:
> 
> In android/res/layout/dialog_download_single_item.xml <https://github.com/mapsme/omim/pull/2740#discussion_r58528792>:
> 
> >  
> >    <View
> > -      android:layout_width="match_parent"
> > -      android:layout_height="1dp"
> > -      android:background="?attr/dividerHorizontal"/>
> > +    android:layout_width="match_parent"
> > +    android:layout_height="1dp"
> > +    android:background="?attr/dividerHorizontal"/>
> attr - лишнее
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/2740/files/8b5f1a4c219311fa724da8ac228f7481878bb4af#r58528792>